Cargo.toml

```
@@ -113,23 +113,23 @@ cookie_store = { version = "0.12", optional = true }
 time = { version = "0.2.11", optional = true }
 
 ## compression
-async-compression = { version = "0.3.0", default-features = false, features = ["stream"], optional = true }
-
+async-compression = { version = "0.3.0", default-features = false, features = ["tokio-03"], optional = true }
```

```suggestion
async-compression = { version = "0.3.6", default-features = false, features = ["tokio-03"], optional = true }
```

Previous versions didn't support the `tokio-03` feature, so requiring 0.3.6 will avoid having projects upgrading from tokio 0.2 without doing a `cargo update` fail to compile because of their Cargo.lock pointing at an older version of async-compression.

---

Thank you for the suggestion!